patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -443,6 +443,8 @@
 +
 +
 +    /* ======================================== FORGE START =====================================*/
++    public void openGui(net.minecraftforge.gui.capability.IGuiProvider provider) { net.minecraftforge.gui.ForgeGuiHandler.openGui(this, field_70170_p, provider); }
```

Comment:
You can expand this to multiple lines so it's clear, it's in the Forge section. As long as you're not importing anything it'll be fine.